docs/_static/css/custom.css

```diff
@@ -12,3 +12,31 @@ pre {
 .rst-content table.docutils td {
   vertical-align: top;
 }
+
+p.remix-link-container {
+    /* Positioned in the top-right corner of the box containing the text snippet that follows it,
+     * with a small margin around it. Should not add any extra space above the snippet box. */
+    position: relative;
+    top: 0.5em;
+    right: 0.5em;
+}
+
+/* Link to Remix IDE shown over code snippets */
+a.remix-link {
+    /* Remix icon. */
+    background:url("https://remix.ethereum.org/assets/img/remixLogo.webp");
```

Comment:
The icon is served directly from https://remix.ethereum.org. I could instead copy it and put it in the repo. It's a small (5 kB) .webp file. Not sure which is better. Either way, we should probably ask them if hotlinking/copying it is fine.

We might also want to swap it out for something else.

Reply:
I'd keep a local copy. Remote URLs can change.

Reply:
Added the image to the PR. We'll probably replace it with a different one though.